geniza/corpus/sitemaps.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date
+
+from django.contrib.sitemaps import Sitemap
+from django.urls import reverse
+
+from geniza.corpus.models import Document
+from geniza.corpus.solr_queryset import DocumentSolrQuerySet
+
+
+def solr_timestamp_to_date(timestamp):
+    """Convert solr isoformat date time string to python date."""
+    # format: 2020-05-12T15:46:20.341Z
+    # django sitemap only includes date, so strip off time
+    yearmonthday = timestamp.split("T")[0]
+    return date(*[int(val) for val in yearmonthday.split("-")])
+
+
+class DocumentSitemap(Sitemap):
+    def items(self):
+        return (
+            DocumentSolrQuerySet()
+            .filter(
+                status="Public"
+            )  # ?: It's not saved as "Document.PUBLIC" which is "P"
+            .only("last_modified", "pgpid")
+        )
```

Comment:
I think you're not getting all of the results — did you check this?

Pretty sure this is  due to a quirk of solr / limitation of parasolr that I haven't figured out the right way to resolve yet (😬 ideas welcome💡 ). By default, Solr only returns 10 rows! When we're iterating over parasolr querysets the logic works fine, but if you just get the first / default set returned by the queryset you only get the default ten. The workaround I've used elsewhere is to just set a ridiculously large maximum number of rows - see e.g. https://github.com/Princeton-CDH/geniza/blob/main/geniza/corpus/admin.py#L275-L280

Reply:
Both `sitemap-documents.xml` and `sitemap-document-scholarship.xml` are showing thousands of results for me locally. I assume when you tested locally you got something different? I'm using Solr 8, are you? Should I just add the `rows=100000` param regardless?